src/t2iapi/combined/combined_requests.proto

```diff
@@ -57,3 +58,15 @@ message SetAlertActivationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4; // the requested OperatingMode value of the operation state
 }
+
+/*
+Request to set the status of an operation mode of an enum metric with the given handle to the requested value and
```

Comment:
maybe change the wording to sth. like this (same for the rpc):

"Request to set the mode of operation represented by an enum metric with the given handle to the requested value ..."

from my point of view this is clearer

Reply:
in the requirement is the wording "operation mode" that's why I think it is better. The similar wording is (was) also used in the function ChangeOperationModeStatusRequest.

Reply:
after review of the new released mpkp standard, the wording should be 'mode of operation'. -> done